.rubocop.yml

```diff
@@ -425,9 +425,15 @@ Rails/FindById:
 Rails/FreezeTime: # new in 2.16
   Enabled: true
 
-Rails/IgnoredColumnsAssignment: # new in 2.17
+Rails/HasManyOrHasOneDependent:
+  Exclude:
+    - 'app/models/user.rb'
+
+Rails/HelperInstanceVariable:
   Enabled: false
-  # we don't use ignored_columns
+
+Rails/IgnoredColumnsAssignment: # new in 2.17
+  Enabled: true
```

Comment:
also, for that matter, this one, and probably more

Reply:
discussed in discord, keeping this one!